### ISSUE TITLE
New York Times changed the wordlist, again

### ISSUE THREAD

**Issue by hackerb9** [bgri/m100LE#46]
According to https://www.nytimes.com/2022/11/07/crosswords/wordle-editor.html, the NYT Wordle Game now has a human editor. I do not quite understand why that was seen as necessary, but the upshot is their wordlist has diverged. I do not yet know if the new wordlist is downloadable. Here's an excerpt from the article:

> [Tracy Bennett, who joined The Times](https://www.nytimes.com/2019/02/20/crosswords/who-made-my-puzzle-bennett.html) as an associate puzzle editor in 2020, will be the editor of Wordle. The game will have a Times-curated word list and will be programmed and tested like the Spelling Bee and the Crossword.
> 
> Wordle’s gameplay will stay the same, and answers will be drawn from the same basic dictionary of answer words, with some editorial adjustments to ensure that the game stays focused on vocabulary that’s fun, accessible, lively and varied.
> 
> The answer list will consist of five-letter words that fit those criteria, with the exception of plural forms of three- or four-letter words that end in “ES” or “S.” That is, the answer will never be FOXES or SPOTS, but it might be GEESE or FUNGI. As the game is currently designed, FOXES or SPOTS can be used as a guess word to help narrow down the answer, but FOXES or SPOTS will not be the answer.

**Comment by bgri** [comment]
Interesting. I guess from an 'importance' perspective having an editor is a good thing. NYT seems to think it's worthwhile to put resources available to the game. Practically that person would only be needed to review wordlists should the meaning of a word change due to cultural reasons. But then, they don't have to scramble to find the person to do the work... they have one on tap. Likely is one of many hats they wear.

As for downloadable, yeah, who knows if they keep the current process, or  change the game to have a 'live' word only. Again, from a resource perspective, it's cheaper for them to have all the computing occur on a remote device vs their hardware. Less back-and-forth too. But it depends on their motivation I guess.

**Comment by hackerb9** [comment]
Looks like the last day we were in sync was November 3rd.

Monday, Oct. 31: APTLY
Tuesday, Nov. 1: PINEY
Wednesday, Nov. 2: INEPT
Thursday, Nov. 3: ALOUD
Friday, Nov. 4: PHOTO
Saturday, Nov. 5: DREAM
Sunday, Nov. 6: STALE

**Comment by bgri** [comment]
Well, I just tried to grab the list using the old methods that worked when it wasn't a NYT product. No joy.

**Comment by hackerb9** [comment]
Well, shoot. I guess M100LE will have to forge its own path forward. Back to the original Josh Wardle list? Continue with the February 2022 NYT list? Something else?

Why we might want something else: I think the biggest part of having an editor is that there were some offensive words that slipped by Josh Wardle, even though he had tried to remove them. For example, on that NYT article there was a comment from a person who had grown up being called various racist epithets and had felt it all come back when the answer to the Wordle recently was SPADE. (I'm not old enough to remember that as a slur, but I can see how you wouldn't want SPADE randomly thrust in front of you without any context, like gardening or playing cards.)

If M100LE has its own word list, should we consider replacing words that the dictionary marks as offensive? I ran a quick script to check for them and there are no more coming up for this year, but there will be about one per year starting next year. 

<details><summary>Click to see potentially offensive words</summary>

2021: (none)
2022: SPADE
2023: QUEER
2024: MAMMY
2025: CHICK
2026: BROAD, PADDY
2027: BUTCH
</details>

**Comment by hackerb9** [comment]
(By the way today's M100LE is your special word! I can't believe NYT changed just in time to miss it.)

**Comment by bgri** [comment]
Good point. I'm ok with drifting away from both of those. Maybe we can substitute words related to this hobby (TANDY, DEBUG, etc) <- not checked to see if they exist already...

re: debug word... heh, yeah, those bums! :) 

Speaking of debug, not sure that code is needed.. I'm working on improving Congrats/Sorry msg and am thinking this code is a fine example of spaghetti code - from the way I was evolving it. If removing sections (like debug) makes it easier, I'm ok with it.
